src/Symfony/ParameterMap.php

```diff
@@ -15,6 +18,15 @@ public function getParameters(): array;
 
 	public function getParameter(string $key): ?ParameterDefinition;
 
+	/**
+	 * @deprecated Will be removed in 2.0
+	 */
 	public static function getParameterKeyFromNode(Expr $node, Scope $scope): ?string;
 
+	//  TODO: Uncomment this in 2.0
+	//	/**
+	//	 * @return array<string>
+	//	 */
+	//	public static function getParameterKeysFromNode(Expr $node, Scope $scope): array;
```

Comment:
I wouldn' be concerned with BC here, I don't think anyone implements this interface in their own code...

Reply:
Great, I reverted the BC commit then :)